lib/LaTeXML/Engine/TeX_Fonts.pool.ltxml

```diff
@@ -27,69 +27,79 @@ use LaTeXML::Package;
 # \fontdimen        iq holds font parameters.
 # \nullfont         iq is a predefined font with no characters.
 
-sub lookupFontinfo {
-  my ($token) = @_;
-  my $defn = LookupDefinition($token);
-  #  return LookupValue(($defn ? ToString($defn) : ToString($token)) . '_fontinfo'); }
-  return LookupValue('fontinfo_' . ($defn ? $defn->getCSName : ToString($token))); }
-
-DefParameterType('FontToken', sub {
-    my ($gullet) = @_;
-    my $token = $gullet->readToken;
+# A Defined Font, or Font Identifier,
+# defined by \font, OR \textfont, \scriptfont,\scriptscriptfont, or even \font itself
+# Actually returns the fontinfo (hash) associated with the font.
+DefParameterType('FontDef', sub {
+    my ($gullet)  = @_;
+    my $readtoken = $gullet->readToken;
+    my $token     = $readtoken;
```

Comment:
I think `$readtoken` is never used. Leftover?

Reply:
yup; thanks!